src/DatePicker/Calendar.js

```diff
@@ -358,7 +355,7 @@ class Calendar extends React.Component {
                   key={this.state.displayDate.toDateString()}
                   minDate={this.props.minDate}
                   maxDate={this.props.maxDate}
-                  onTouchTapDay={this.handleTouchTapDay}
+                  onDayTouchTap={this.handleTouchTapDay}
```

Comment:
I changed this because CalendarMonth had `onDayTouchTap: PropTypes.func,` so I kept it to be the same as I was getting issue while changing dates after this. 

I thought the naming convention was changed.